WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -119,9 +117,9 @@ public HistoryViewModel(WalletViewModel walletViewModel, IObservable<Unit> updat
 					width: new GridLength(1, GridUnitType.Star)),
 
 				// Incoming
-				new PrivacyTextColumn<HistoryItemViewModelBase>(
+				new TemplateColumn<HistoryItemViewModelBase>(
 					"Incoming (BTC)",
-					x => x.IncomingAmount?.ToFormattedString(),
+					new FuncDataTemplate<HistoryItemViewModelBase>((node, ns) => new PrivacyContentControl { Content = node.IncomingAmount, NumberOfPrivacyChars = 9 }, true),
```

Comment:
The PrivacyTextColumn was specifically used to improve history table performance, especially scrolling perf.